### DISCUSSION TITLE
Where do I post issues with the Save Tab Groups extension?

### DISCUSSION THREAD

**Discussion by NylaTheWolf** [sienori/Tab-Session-Manager#882] [Q&A] [answered]
If I have an issue with the [Save Tab Groups extension](https://github.com/sienori/Save-Tab-Groups-for-Tab-Session-Manager) for this, would I post it in the Issues section of this repository or on the Save Tab Groups' repository? [I posted my issue 18 days ago](https://github.com/sienori/Tab-Session-Manager/issues/869#issue-996395090) but I never got a response, so I'm wondering if I did something wrong?

**Comment by sienori** [comment, marked as answer]
Sorry my late reply! This repository is the correct place to submit the issue about Save Tab Groups extension.

**Comment by Robidimaio** [comment]
You're very lucky to have received an answer, or do you pay support before trying this useful extension?
There are interesting questions that have not been answered for quite a few months, including mine.

I'm desperate to try both TSM and STG but "When you install it, you will see a new floppy disk icon on the toolbar; this is how you access Tab Session Manager." is not my case. 
There is no icon at all in Chrome 98.0.4758.82 - 64 bit / Windows 10 - 19044.1415 build.

I tried to uninstall/reinstall the extension after deleting (by mistake?) a few folders with very strange names both on my PC and Google Drive and discovered afterward that those folders were created by TSM. If I knew before, I wouldn't have deleted them of course.

I'm using about 20 Chrome pages with +/- 1.500 tabs divided into +/- 40 Groups.
Every time I restart Chrome many windows appear, suggesting to save .json files, and I regularly save them. 

Tab Session Manager 6.11.1
Save Tab Groups for Tab Session Manager 1.0.2

**Reply by millermk1717** [reply]
Have you pinned TSM to the toolbar first? Do you use an extension manager?
